raiden/transfer/channel.py

```diff
@@ -113,6 +113,18 @@
 SendUnlockAndMerkleTree = Tuple[SendBalanceProof, MerkleTreeState]
 
 
+def get_sender_expiration_threshold(lock: HashTimeLockState) -> BlockNumber:
+    return BlockNumber(
+        lock.expiration + DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS,
```

Comment:
why sender and receiver are using the same expression?

Reply:
Oh shit! i was wondering why some tests are failing